gitbook-docs/market/user-guides/proof-producer.md

```diff
@@ -1,82 +1,71 @@
-# Proof Producer
+# Proof producer
 
 {% hint style="info" %}
-Ensure you have done the [Authentication](sign-up.md) setup before progressing
+Make sure you've performed the [authentication setup](sign-up.md) before continuing.
 {% endhint %}
 
-## Submit Ask order
+## Submit a proposal
```

Comment:
I wonder why we submit requests and proposals but publish statements. And all of them we `push` with CLI.
Don't change it for now, but let's discuss. :)